crates/uv/src/settings.rs

```diff
@@ -46,8 +46,12 @@ impl GlobalSettings {
         Self {
             quiet: args.quiet,
             verbose: args.verbose,
-            color: if args.no_color {
+            color: if args.no_color || std::env::var_os("NO_COLOR").is_some() {
                 ColorChoice::Never
+            } else if std::env::var_os("FORCE_COLOR").is_some()
+                || std::env::var_os("CLICOLOR_FORCE").is_some()
+            {
+                ColorChoice::Always
```

Comment:
NO_COLOR says it should not be an empty string.  Same with FORCE_COLOR.  CLICOLOR *says* it should just be set but the example Python code checks that its non-empty.

Reply:
Good point, I was looking at https://github.com/python/cpython/blob/3.13/Lib/_colorize.py#L53 too much

Reply:
affba38815b1b4535dab9a6211f211a24c316f61, is this more along the lines of what you were thinking? @epage 

Thanks for the review btw.